settings.gradle

```diff
@@ -45,3 +45,20 @@ rootProject.children.each {
     it.name = it.name + "_" + scalaVersion
   }
 }
+
+/**
+ * Skips execution of all tasks for the project 'samza-test'.
+ * To include `samza-test`, initiate gradle build by setting the property 'includeSamzaTest'.
+ * To build with samza-test: './gradlew clean build -PincludeSamzaTest'
+ */
+def propertyName = "includeSamzaTest"
+def projectName = "samza-test"
```

Comment:
> There will no benefit in building locally samza-test if it’s not going to be run locally (since it just contains tests, doesn't have src files).

It contains the test source files, which should be compiled as part of the build to detect compilation errors/API changes.